### PR TITLE
Chore/update schema

### PR DESCRIPTION
**Pull request by Project-FAB**
Updated prism.schema based on ERD and add getUserByTeam

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -11,43 +11,73 @@ datasource db {
...
+  updated_at        DateTime          @default(now())
   schedule          String[]
-  position          String[]
-  orgChart          org_chart?
-  timeLogs          time_logs[]
-  userprojects      UserProjects[]
-  userclients       UserClients[]
-  todoList          todo_list[]
-  note              Notes[]
-  roleType          RoleTypeEnum
```
Comment (420ku) [inline prisma/schema.prisma:32]: roleType should be enum so user can only choose mentor/intern in FE

File: prisma/schema.prisma [hunk 2]
```diff
@@ -11,43 +11,73 @@ datasource db {
...
-  userType          UserTypeEnum   @default(USER)
```
Comment (420ku) [inline prisma/schema.prisma:33]: this is to classify superAdmin from users. if user creates account their userType would be USER by default

File: prisma/schema.prisma [hunk 3]
```diff
@@ -59,66 +89,93 @@ model time_logs {
...
+  id               String     @id @default(uuid()) @map("_id")
+  title            String
+  subheader        String
+  full_description String
+  tags             TodoTags[]
+  users_id         String     @unique
+  users            Users      @relation(fields: [users_id], references: [id])
+  project_id       String?    @unique
+  projects         Projects?  @relation(fields: [project_id], references: [id])
+  prio_level       Int
```
Comment (420ku) [inline prisma/schema.prisma:102]: i suggest this for enum type

File: prisma/schema.prisma [hunk 4]
```diff
@@ -11,43 +11,73 @@ datasource db {
...
+  todo_list         todo_list?
+  UserProjects      UserProjects[]
+  clients           UserClients[]
+  Notes             Notes[]
 
   @@unique([email_address])
 }
 
+model Work_Experience {
+  word_id    String   @id @default(uuid()) @map("_id")
```
Comment (420ku) [inline prisma/schema.prisma:47]: i suggest change field name to id

File: prisma/schema.prisma (hunk 3 above)
Comment (Project-FAB) [reply prisma/schema.prisma:102]: okay will do enum for this

File: prisma/schema.prisma (hunk 1 above)
Comment (Project-FAB) [reply prisma/schema.prisma:32]: okay.

**Comment by Project-FAB** [comment]
updated with ref 63fd3fb

**Comment by Project-FAB** [comment]
updated with ref 1b77c83

**Pull request merged**